src/App.jsx

```diff
@@ -14,33 +19,88 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+function getPraparedGoods(goods, sortField) {
```

Comment:
```suggestion
function sortGoods(goods, sortField) {
```
`sortGoods` describes this function better, as you only sort them